docs/getting-started/assets/orders-service-openapi.yaml

```diff
@@ -0,0 +1,140 @@
+openapi: 3.0.1
+info:
+  title: Orders service and Events API
```

Comment:
Do you want to apply [this](https://github.com/kyma-project/kyma/pull/9296/#discussion_r483098183) here too?